Microsoft.Toolkit/Extensions/ArrayExtensions.cs

```diff
@@ -10,22 +10,21 @@
 namespace Microsoft.Toolkit.Extensions
 {
     /// <summary>
-    /// Helper extension methods for Arrays.
+    /// Helpers for working with arrays.
     /// </summary>
     public static class ArrayExtensions
     {
         /// <summary>
-        /// Fills in values of a multi-dimensional rectangular array to specified value based on the position and size given.
-        /// Ranges given outside the bounds of the array will fill in as much as possible and ignore elements that should appear outside it.
```

Comment:
Why is this removed?

Reply:
I think the positions being "ignored" implies that exceptions won't be thrown. I've changed up the second line to include more information; what do you think now?